src/data_source/storage/sql.rs

```diff
@@ -2219,8 +2219,10 @@ where
                 FROM header AS h
                 JOIN payload AS p ON
                     p.height = h.height
-                ORDER BY h.height DESC
-                LIMIT 50",
+                WHERE
+                    h.height IN (SELECT height FROM header ORDER BY height DESC LIMIT 50)
+                ORDER BY h.height ASC 
```

Comment:
I'm thinking it may be better to avoid the sub-query/`IN` predicate, instead just select by `height DESC LIMIT 50` and then reverse the order in memory. I'm not sure though.

Reply:
I considered doing this... But for the sake of ease, I opted for the query since it only requires the change in a single place.  Though I am open to reversing it in memory as well.

Reply:
We can leave it unless/until it causes problems. It probably won't